extension/rusheye/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+        <groupId>org.jboss.arquillian.graphene</groupId>
+        <artifactId>graphene-parent</artifactId>
+        <version>2.3.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+   </parent>
+   <artifactId>arquillian-graphene-rusheye</artifactId>
+   <packaging>pom</packaging>
+   <name>Graphene Extension: RushEye - Parent</name>
+   <description>An extension for Arquillian Graphene for visual validation</description>
+   <modules>
+      <module>arquillian-graphene-rusheye-impl</module>
+      <module>arquillian-graphene-rusheye-test</module>
+   </modules>
+   <properties>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <version.org.jboss.arquillian>1.1.13.Final</version.org.jboss.arquillian>
+      <version.org.jboss.arquillian.drone>2.1.1</version.org.jboss.arquillian.drone>
+      <version.org.jboss.arquillian.graphene>${project.version}</version.org.jboss.arquillian.graphene>
+   </properties>
+   <dependencies>
+        <dependency>
+            <groupId>org.jboss.arquillian.graphene</groupId>
+            <artifactId>graphene-webdriver</artifactId>
+            <version>${version.org.jboss.arquillian.graphene}</version>

```

Comment:
the property `version.org.jboss.arquillian.graphene` is redundant - use here the `${project.version}`